src/main/java/by/exadel/internship/dto/User.java

```diff
@@ -0,0 +1,29 @@
+package by.exadel.internship.dto;
+
+import by.exadel.internship.dto.enums.UserRole;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+
+import java.time.LocalDateTime;
+import java.util.List;
+import java.util.UUID;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+public class User {
```

Comment:
As far as I understand, your entities will have the same names, so It's good practice to add  'DTO' postfix to the class name. Please, do it for all DTOs.

Reply:
done